cterasdk/edge/shares.py

```diff
@@ -58,6 +58,24 @@ def add(self,
         except Exception as error:
             logging.getLogger().error("Share creation failed.")
             raise CTERAException('Share creation failed', error)
+            
+    def set_acl(self, name, acl):
+        """
+        Set a network share's access control entries.
+        
+        :param name: the name of a network share
+        :param acl: a list of 3-tuple access control entries
+        :type name: str
+        :type acl: list[tuple(str, str, str)]
+        
+        .. warning: this method will override the existing access control entries
+        """
+        param = []
+        for entry in acl:
```

Comment:
I know it is not specific to this change and I am making this change in other places as well. But, you should use namedtuples for the ACL. See https://github.com/CTERA-Networks/ctera-python-sdk/blob/ygal/create_new_doc/cterasdk/core/zones.py#L190 for reference